Automattic/jetpack
# Tock Block: Implement settings with a dropdown

This is an alternative version of #31338. 

## Proposed changes:


On top of the changes in #31338, this PR:

* Swaps to using the Dropdown component to make the implementation of the popover more standard
* Adds some extra logic to reset the input when the block or dropdown loses focus
* Removes the notice in the editor to prevent things jumping around.

### Other information:


## Jetpack product discussion
p1HpG7-n0W-p2#comment-63988

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:





This is the same as #31303

You will still see a notice in the front end if you have `edit_posts` permissions

<img width="700" alt="image" src="https://github.com/Automattic/jetpack/assets/96462/8668a9c9-4369-4137-bb75-0f228e461cd2">

I think this is safe as it won't be appearing and disappearing.
